Assets/HoloToolkit-Preview/QRTracker/Scripts/QRCode.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+
+using System.Collections.Generic;
+using UnityEngine;
+
+#if WINDOWS_UWP
+
+using Windows.Perception.Spatial;
+
+#endif
+namespace HoloToolkit.Unity.QRTracking
+{
+    [RequireComponent(typeof(HoloToolkit.Unity.SpatialGraphCoordinateSystem))]
+    public class QRCode : MonoBehaviour
+    {
+        public QRCodesTrackerPlugin.QRCode qrCode;
+        private GameObject qrCodeCube;
+
+        public float PhysicalSize;
+        public string CodeText { get; set; }
+
+        private TextMesh QRID;
+        private TextMesh QRText;
+        private TextMesh QRVersion;
+        private TextMesh QRTimeStamp;
+        private TextMesh QRSize;
+        private GameObject QRInfo;
+
+        private long lastTimeStamp = 0;
+
+        // Use this for initialization
+        void Start()
+        {
+            PhysicalSize = 0.1f;
+            CodeText = "Dummy";
+            if (qrCode == null)
+            {
+                throw new System.Exception("QR Code Empty");
+            }
+
+            PhysicalSize = qrCode.PhysicalSizeMeters;
+            CodeText = qrCode.Code;
+
+            qrCodeCube = gameObject.transform.Find("Cube").gameObject;
```

Comment:
Might be a better way to look for the transforms.

Reply:
Agreed, should really be assigned to a property for assigning in the editor.

Reply:
This script is example visualization just for a particular prefab. Not intended to be used for any other objects.